Makefile

```diff
@@ -9,6 +9,7 @@ KUBECONFIG := $(shell pwd)/.kubeconfig
 
 # Default values
 DOCKER_COMPOSE_OVERRIDE=
+DOCKER_COMPOSE=$(shell if command -v docker-compose > /dev/null; then echo 'docker-compose'; else echo 'docker compose'; fi)
```

Comment:
Can you explain this condition? Both branches return `docker-compose` in the end?

Reply:
Mind the dash. You either geht `docker-compose` (docker compose stand-alone exeuctable) or `docker compose` (docker compose plugin).

Reply:
Ah. Now I can see the dash. Thanks for explanation. 🙈